mindsdb/libs/phases/data_extractor/data_extractor.py

```diff
@@ -113,7 +119,7 @@ def _validate_input_data_integrity(self):
         """
         :return:
         """
-        if self.transaction.input_data.data_frame.shape[0] <= 0:
+        if len(self.transaction.input_data.data_frame) <= 0:
```

Comment:
why this change?

Reply:
Because initially dask was being used in this part (switched back to pandas in the last commit) and it doesn't support `.shape`